config/config.go

```diff
@@ -1043,9 +1044,21 @@ func parseConfig(contents []byte) (*ast.Table, error) {
 		if ok {
 			envVal = escapeEnv(envVal)
 			contents = bytes.Replace(contents, parameter[0], []byte(envVal), 1)
+		} else {
+			missingEnvVars = append(missingEnvVars, string(envVar))
 		}
 	}
 
+	// Report all missing environment variables to the user
+	if len(missingEnvVars) > 0 {
+		errorMsg := "failed to find the configured environment variables:"
```

Comment:
```suggestion
		errorMsg := "environment variable(s) not set:"
```

what do you think about saying they are not set, rather than failed to find them?

Reply:
Good suggestion!